### ISSUE TITLE
[Feature Request] Option to set number of sessions or duration before starting the timer

### ISSUE THREAD

**Issue by x51mon** [adrcotfas/goodtime#209]
I think it would be nice to have an option to set either a a number of work sessions or an amount of time, before starting the timer, and then getting a notification when those are used up. That way it would be easier to dedicate a certain amount of time to a project or task when working on several things at the same time.
